rootfs/scheduler/__init__.py

```diff
@@ -770,8 +772,9 @@ def _create_service(self, name, app_name, app_type, data={}, **kwargs):
         image = kwargs.get('image')
         try:
             image = self.registry + '/' + image
+            repo = image.split(":")
             # image already includes the tag, so we split it out here
-            docker_cli.pull(image.rsplit(':')[0], image.rsplit(':')[1])
+            docker_cli.pull(repo[0]+":"+repo[1], tag=repo[2], insecure_registry=True)
```

Comment:
I copied this line from publish_release thought a best practice 
https://github.com/deis/workflow/blob/master/rootfs/registry/dockerclient.py#L54